test/unit/remote/index.js

```diff
@@ -217,9 +217,8 @@ describe('remote.Remote', function () {
         }
       }
       await this.remote.addFileAsync(doc)
-      should.exist(doc.remote._id)
-      should.exist(doc.remote._rev)
-      should.exist(doc._deleted)
+      should(doc._deleted).be.true()
+      should(doc).not.have.property('remote')
```

Comment:
Oh my... File could not be uploaded so `doc` shouldn't have a `remote` afterwards.
:+1: good catch.